lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,91 @@
+require 'coffeelint'
+
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner < Runner
+    CLI_OPTION = 'g'
+    CLI_FULL_OPTION = 'coffee-lint'
+    DESCRIPTION = 'Run coffeescript style guide validation'
+    COMMAND_NAME = 'coffeelint'
+    FAILURE_REGEXP = /(?<issues>(WARN|ERROR))/
+
+    private
+
+    def run_command
+      @stdout.puts(exec_msg) unless quiet?
+      lint_results.each do |filename, result|
+        next if result.empty?
+        yield "\n#{filename}\n#{MessageFormatter.new(result).format}"
+      end
+      @stdout.puts(stats_msg) if stats?
+    end
+
+    def lint_results
+      lint_results = {}
+      @options.files.split(' ').each do |filename|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.